python/ambassador_diag/diagd.py

```diff
@@ -465,6 +467,11 @@ def check_ready():
 def show_overview(reqid=None):
     app.logger.debug("OV %s - showing overview" % reqid)
 
+    # getting elements in the `tvars_cache` will make sure eviction happens on `max_age_seconds` TTL
```

Comment:
Maybe do this in the detail case as well? or even in the `standard_handler`...

Reply:
Great suggestion, thanks! I moved the eviction to the `standard_handler`.